tools/deb/test/run_gce_test.sh

```diff
@@ -235,7 +237,7 @@ spec:
   ports:
```

Comment:
let's make the switch to istioctl register here
(see the end of https://github.com/istio/istio/blob/rawvm-register-3/samples/rawvm/README.md for an example)